history.rst

```diff
@@ -5,6 +5,24 @@
 OMERO version history
 =====================
 
+5.6.2 (July 2020)
+-----------------
+
+This release adds installation documentation for server and web on
+CentOS 8 and Ubuntu 20.04, and it removes installation instructions
+for Ubuntu 16.04.
+
+This version has been tested with:
+
+- omero-blitz 5.5.7
+- omero-gateway-java 5.6.4
+- omero-py 5.7.1
+- omero-web 5.7.0
+- omero-dropbox 5.6.2
```

Comment:
Rfe it might be useful to link to the components changelogs or maybe summarize the major features.

Reply:
Links to the changelogs are in https://github.com/ome/ome-documentation/blob/develop/omero/changelog.rst